docs/monitor-kubernetes.md

```diff
@@ -22,52 +14,94 @@ To set up monitoring of Kubernetes, you need the following:
-
-1. Get the PMM API key. The key must have the role "Admin".
+    Get the PMM API key: 
 
-    === "From PMM UI" 
+    === ":material-view-dashboard-variant: From PMM UI" 
 
         [Generate the PMM API key](https://docs.percona.com/percona-monitoring-and-management/details/api.html#api-keys-and-authentication){.md-button} 
 
-    === "From command line"
+    === ":material-console: From command line"
 
         You can query your PMM Server installation for the API
         Key using `curl` and `jq` utilities. Replace `<login>:<password>@<server_host>` placeholders with your real PMM Server login, password, and hostname in the following command:
 
         ``` {.bash data-prompt="$" }
-        $ API_KEY=$(curl --insecure -X POST -H "Content-Type: application/json" -d '{"name":"operator", "role": "Admin"}' "https://<login>:<password>@<server_host>/graph/api/auth/keys" | jq .key)
+        $ API_KEY=$(curl --insecure -X POST -H "Content-Type: application/json" -d {"name":"operator", "role": "Admin"}' "https://<login>:<password>@<server_host>/graph/api/auth/keys" | jq .key)
+            ```
+
+        !!! note
+
+            The API key is not rotated. 
+
+## Install the Victoria Metrics Kubernetes monitoring stack
+
+1. To install the Victoria Metrics Kubernetes monitoring stack with the default parameters, use the quick start command. Replace the following placeholders with your values:
+
+    * `API-KEY` - The [API key of your PMM Server](#get-the-pmm-server-api-key)
+    * `PMM-SERVER-URL` - The URL to access the PMM Server 
+    * `UNIQUE-K8s-CLUSTER-IDENTIFIER` - Identifier for the Kubernetes cluster. It can be the name you defined during the cluster creation.    
+
+      You should use a unique identifier for each Kubernetes cluster. The use of the same identifer for more than one Kubernetes cluster will result in the conflicts during the metrics collection.    
+
+    * `NAMESPACE` - The namespace where the Victoria metrics Kubernetes stack will be installed. If you haven't created the namespace before, it will be created during the command execution.     
+
+      We recommend to use a separate namespace like `monitoring-system` for the same.    
+
+      ```{.bash data-prompt="$" }
+      $ curl -fsL  https://raw.githubusercontent.com/Percona-Lab/k8s-monitoring/main/vm-operator-k8s-stack/quick-install.sh | bash -s -- --api-key <API-KEY> --pmm-server-url <PMM-SERVER-URL> --k8s-cluster-id <UNIQUE-K8s-CLUSTER-IDENTIFIER> --namespace <NAMESPACE> 
+      ```    
+
+    !!! note    
+
+        The Prometheus node exporter is not installed by default since it requires privileged containers with the access to the host file system. If you need the metrics for Nodes, enable the Prometheus node exporter by setting the `prometheus-node-exporter.enabled` flag in the Victoria Metrics Helm chart to `true`. Refer to the [Install manually](install-the-victoria-metrics-kubernetes-monitoring-stack-manually) section.
```

Comment:
For the quick install, user can just use another flag `--node-exporter-enabled` 
which in turn translates to 

```
curl -fsL  https://raw.githubusercontent.com/Percona-Lab/k8s-monitoring/main/vm-operator-k8s-stack/quick-install.sh | bash -s -- --api-key <API-KEY> --pmm-server-url <PMM-SERVER-URL> --k8s-cluster-id <UNIQUE-K8s-CLUSTER-IDENTIFIER> --namespace <NAMESPACE> --node-exporter-enabled
```